extensions/indexes/lucene/src/org/exist/indexing/lucene/LuceneIndexWorker.java

```diff
@@ -487,6 +488,13 @@ protected void setOptions(Properties options, CommonQueryParserConfiguration par
             else
                 parser.setMultiTermRewriteMethod(MultiTermQuery.CONSTANT_SCORE_BOOLEAN_QUERY_REWRITE);
         }
+        option = options.getProperty(OPTION_LOWERCASE_EXPANDED_TERMS);
+        if (option != null) {
+            if (option.equalsIgnoreCase("yes"))
```

Comment:
I took the model from
        option = options.getProperty(OPTION_FILTER_REWRITE);
        if (option != null) {
            if (option.equalsIgnoreCase("yes"))
                parser.setMultiTermRewriteMethod(MultiTermQuery.CONSTANT_SCORE_FILTER_REWRITE);
            else
                parser.setMultiTermRewriteMethod(MultiTermQuery.CONSTANT_SCORE_BOOLEAN_QUERY_REWRITE);
        }
I hardly dare to change what you all wrote. :)

Anyway, any improvement is welcome.